app/Helpers/MSGraphAPI/UserByApp.php

```diff
@@ -0,0 +1,216 @@
+<?php
+namespace App\Helpers\MSGraphAPI;
+
+
+use Microsoft\Graph\Graph;
+use GuzzleHttp;
+use Illuminate\Support\Str;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+
+class UserByApp {
+    protected $employeeNumberProperty ;
+    protected $descriptionProperty ;
+    protected $mailProperty ;
+    protected $RoleAdminId;
+    protected $RoleUserId;
+
+    public function __construct()
+    {
+        $extension_app_id = Str::remove('-', config('services.azure.extension_app_id'));
+        $this->employeeNumberProperty = 'extension_' . $extension_app_id . '_employeeNumber';
+        $this->descriptionProperty = 'extension_' . $extension_app_id . '_description';
+        $this->mailProperty = 'extension_' . $extension_app_id . '_mail';
+    }
+
+
+
+    public function getUsers($topN=null,$searchProperty=null,$searchValue=null,$checkAdmin=False) {
```

Comment:
Functies niet langer dan 16 regels code doen?
Zorg ervoor dat je stukken logica wegschrijft naar een functie met duidelijke naam.

Is het voor mij leesbaarder wat er gebeurt en helpt ook bij toekomstige wijzigingen